bitnami/thanos/values-production.yaml

```diff
@@ -1119,10 +1138,10 @@ minio:
   ## MinIO credentials
   ##
   accessKey:
-    password: ""
+    password: ''
   secretKey:
-    password: ""
+    password: ''
 
   ## Default MinIO buckets
   ##
-  defaultBuckets: "thanos"
+  defaultBuckets: 'thanos'
```

Comment:
Is there a reason this is changed here, but not in the regular `values.yaml`?

Reply:
Sorry, I formatted only `values-production.yaml`, fixed now! ;)